compiler/gentree.cpp

```diff
@@ -436,6 +436,13 @@ VertexPtr GenTree::get_expr_top(bool was_arrow) {
       next_cur();
       break;
     }
+    case tok_int_const_sep: {
+      std::string val = static_cast<string>(cur->str_val);
+      remove_underscores(val);
```

Comment:
Since token is "_sep" (separators?), maybe call remove_underscores as remove_separators?

Reply:
Done.